### PR TITLE
Add RtpEncodingParameters.maxBitrate.

### PR DESCRIPTION
**Pull request by pthatcherg**

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -2763,6 +2763,14 @@
               bandwidth.
             </p>
           </dd>
+
+          <dt>unsigned long maxBitrate</dt>
+          <dd>
+            <p>
+              Indicates the maximum bitrate that can used to send this
```
Comment (alvestrand) [inline webrtc.html]: Nit: "can be used"

Query: How does it interact with other bitrates (per-transport or per-session), and where should text describing such interaction go?

Comment (pthatcherg) [inline webrtc.html]: Fixed the typo, and added a comment about how per-transport or per-session bandwidth limits may further constrain the encoding.

**Comment by pthatcherg** [comment]
I added a TODO for a bandwidth definition.

**Comment by pthatcherg** [comment]
I did a rebase when I probably should have done a merge.  Now it's all ugly.  I made a cleaned-up copy here: https://github.com/w3c/webrtc-pc/pull/268

**Comment by alvestrand** [comment]
Closed in favor of #268.

**Pull request closed without merging**